Pod/SQL/GDGHasManyRelation.m

```diff
@@ -103,8 +107,19 @@ - (void)fill:(NSArray<GDGEntity *> *)entities fromQuery:(SQLEntityQuery *)query
 
 - (void)hasBeenSetOnEntity:(GDGEntity *)entity
 {
-	for (GDGEntity *owned in [entity valueForKey:self.name])
-		[owned setValue:entity.id forKey:self.foreignProperty];
+	if (entity.id != nil)
+		for (GDGEntity *owned in [entity valueForKey:self.name])
+			[owned setValue:entity.id forKey:self.foreignProperty];
+}
+
+- (void)save:(GDGEntity *)entity
+{
+	for (GDGEntity *related in [entity valueForKey:self.name])
+	{
+		[related setValue:entity.id forKey:self.foreignProperty];
+		if (![related save:nil])
+			NSLog(((SQLEntityMap *)self.relatedMap).table.databaseProvider.database.lastErrorMessage, nil);
```

Comment:
What do you think about use the `NSError**` here, pass it to inner save method e remove the NSLog?